src/index.ts

```diff
@@ -61,6 +67,21 @@ export function form<Decl extends FormDeclaration<any>>(
     }: Sources<Decl>): Sinks<Decl> {
         const touchedKeys = new Set<keyof Decl>();
 
+        let customSubmission$: Stream<KeyboardEvent> | null = null;
+        if (options && options.customSubmission) {
```

Comment:
Instead of checking the presence of the option, how about providing the default value for `options`?

Reply:
:white_check_mark: https://github.com/herp-inc/cyclic-form/pull/2/commits/6f24388db43342b60cfefa1048cf8d2ddbd39848